gitlab/transformer_functions.py

```diff
@@ -0,0 +1,196 @@
+import base64
+
+from external_plugins.gitlab import default as DEFAULT 
+from agilitysync.external_lib.restapi import ASyncRestApi
+
+fields = [
+    {
+    "required": True,
+    "disabled field": False,
+    "customfield":False,
+    "raw_title": "title",
+    "title":"title",
+    "type":"title",
+    "IsMultivalue" : False
+},
+{
+    "required": False,
+    "disabled field": False,
+    "customfield":False,
+    "raw_title": "description",
+    "title":"description",
+    "type":"description",
+    "IsMultivalue" : False
+},
+
+{
+    "required": False,
+    "disabled field": False,
+    "customfield":False,
+    "raw_title": "labels",
+    "title":"labels",
+    "type":"labels",
+    "IsMultivalue" : False
+},
+{
+    "required": False,
+    "disabled field": False,
+    "customfield":False,
+    "raw_title": "weight",
+    "title":"weight",
+    "type":"weight",
+    "IsMultivalue" : False
+},
+
+
+]
+def ticfields(self):
+    return fields
+def connect(instance_details):
+   return ASyncRestApi(instance_details['url'],headers={
+                            "authorization": "Bearer {}".format(instance_details["token"]),
+                            "Accept": "application/json",
+                            "Content-Type": "application/json"
+                            })
+
+
+
+
+
+def check_connection(instance,instance_details):
+    instance_path = "repos"
```

Comment:
constants have to be in full capital. Move this instance_path to top like - REPOS_ENDPOINT = "repos"